vm/actor/src/builtin/puppet/mod.rs

```diff
@@ -0,0 +1,201 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use num_derive::FromPrimitive;
+use num_traits::FromPrimitive;
+
+use crate::check_empty_params;
+use address::Address;
+use encoding::{tuple::*, Cbor, CodecProtocol, Error};
+use ipld_blockstore::BlockStore;
+use num_bigint::bigint_ser;
+use runtime::{ActorCode, Runtime};
+use serde::{Deserialize, Serialize};
+use vm::{ActorError, ExitCode, MethodNum, Serialized, TokenAmount, METHOD_CONSTRUCTOR};
+
```

Comment:
Lol i know this actor does nothing, but for the sake of consistancy with the other actors, could you add the commit hash this was ported from?